pcap/pcap.go

```diff
@@ -89,6 +89,29 @@ int pcap_set_rfmon(pcap_t *p, int rfmon) {
 #define gopacket_time_secs_t time_t
 #define gopacket_time_usecs_t suseconds_t
 #endif
+int pcap_wait(pcap_t *p, int usec) {
+	fd_set fds;
+	int fd;
+	struct timeval tv;
+
+	fd = pcap_get_selectable_fd(p);
```

Comment:
According to the man pages, ` pcap_get_selectable_fd() is not available on Windows.`

Folks do use this for winpcap, so we'll need to hande that eventuality.